wye/templates/email_messages/contactus/message.html

```diff
@@ -0,0 +1,5 @@
+Contact Name : {{contact_name}}
```

Comment:
Can we add more context here
Saying
Hi All,
   We have received feedback from <name> . Requesting to respective Regional or Admin members to reply and inform all on action taken.
+Topic : {{query_topic}}
+Mobile : {{mobile}}
+Content : {{content}}